src/samples/basic.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>OpenStreetMap Americana - Basic Demo</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://zelonewolf.github.io/openstreetmap-americana/favicon.ico"
+    />
+    <meta
+      name="viewport"
+      content="initial-scale=1,maximum-scale=1,user-scalable=no"
+    />
+    <script src="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.js"></script>
+    <link
+      href="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.css"
+      rel="stylesheet"
+    />
+    <style>
+      body {
+        margin: 0;
+        padding: 0;
+      }
+      #map {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        width: 100%;
+      }
+      #attribution-logo {
+        position: absolute;
+        bottom: 10px;
+        z-index: 100;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="map"></div>
+    <div id="attribution-logo"></div>
```

Comment:
What is this?

Reply:
Ooh yeah, that's a leftover container from when we had to display a maptiler logo in order to use their tileserver.  We can safely scrub that (e1c7162)